lua/entities/gmod_wire_expression2/base/preprocessor.lua

```diff
@@ -43,12 +43,20 @@ function PreProcessor:HandlePPCommand(comment)
 end
 
 function PreProcessor:FindComments(line)
+	local isinput = not self.blockcomment and not self.multilinestring and line:match("^@inputs") ~= nil
```

Comment:
Does @inputs have to be the start of the line?

Reply:
Yes